src/transformers/utils/import_utils.py

```diff
@@ -36,6 +36,7 @@
 logger = logging.get_logger(__name__)  # pylint: disable=invalid-name
 
 
+# TODO: This doesn't work for all packages (`bs4`, `faiss`, etc.) Talk to Sylvain to see how to do with it better.
```

Comment:
a possibility could be to split the `pkg_name` arg in two. one for find_spec and the other for metadata, with the default being equal.

Reply:
@sgugger Do you have any comment regarding the changes I made in this PR?

Reply:
I think it's fine as it is. No need to make `_is_package_available` more complex since the alternative takes one line.